addon/components/tool-tipster.js

```diff
@@ -137,10 +137,10 @@ export default Component.extend({
 
   willDestroyElement() {
     this._super(...arguments);
-    if (this.$().data('tooltipster-ns')) {
```

Comment:
looks like the jQuery codemod isn't actually all that good :(

Reply:
Hehe yeah this is wrapper on top of jquery plugin, but it's not maintained anymore.